pyccel/codegen/printing/ccode.py

```diff
@@ -282,13 +282,45 @@ def _format_code(self, lines):
         return self.indent_code(lines)
 
     def _flatten_list(self, irregular_list):
-        if isinstance(irregular_list, (PythonList, PythonTuple)):
+        if isinstance(irregular_list, (PythonList, PythonTuple, list)):
             f_list = [element for item in irregular_list for element in self._flatten_list(item)]
             return f_list
         else:
             return [irregular_list]
 
     #========================== Numpy Elements ===============================#
+
+    def varCpy(self, lhs, rhs, expr, pad=""):
+        """ generates the 'memcpy' line needed to cpy a 'Variable/ndarray' to another
```

Comment:
you should recheck the docs, because I don't see where memcpy is generated.

Reply:
I updated the docs, and I removed a uselss parameter.